inc/render/class-stripe-checkout-block.php

```diff
@@ -39,6 +39,10 @@ public function render( $attributes ) {
 			if ( false !== $status ) {
 				if ( 'success' === $status ) {
 					$message = isset( $attributes['successMessage'] ) ? wp_kses_post( $attributes['successMessage'] ) : __( 'Your payment was successful. If you have any questions, please email orders@example.com.', 'otter-blocks' );
+
+					if ( has_action( 'otter_blocks_stripe_checkout_success' ) ) {
```

Comment:
Why do you check if there's an action before running do_action?

Reply:
To make sure we do not have problems with people who do not have Otter Pro since the action is only registered in Pro

Reply:
Yea, it's fine. Other people can extend it if they want. It's anyway redundant. If someone adds an action to that hook then it will run regardless.